cmd/dagger/cmd/do.go

```diff
@@ -85,13 +85,9 @@ var doCmd = &cobra.Command{
 		}
 
 		actionFlags := getActionFlags(action)
-
+		actionFlags.Parse(args)
 		cmd.Flags().AddFlagSet(actionFlags)
 
-		cmd.Flags().ParseErrorsWhitelist = pflag.ParseErrorsWhitelist{
-			UnknownFlags: false,
-		}
-		err = cmd.Flags().Parse(args)
 		if err != nil {
```

Comment:
Does the error from `err = actionFlags.Parse(args)`?

`cmd.Flags().AddFlagSet(actionFlags)` may be after this err handler?

Reply:
> Does the error from `err = actionFlags.Parse(args)`?

We shouldn't check the error here because `actionFlags.parse(args)` will most likely return an error since it will complain about `not defined arguments`. Hence, using a `ContinueOnError` parsing strategy.